pkg/virt-api/webhooks/validating-webhook/admitters/migrationpolicy-admitter_test.go

```diff
@@ -45,8 +41,6 @@ import (
 )
 
 var _ = Describe("Validating MigrationPolicy Admitter", func() {
-	config, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
-
 	var ctrl *gomock.Controller
 	var virtClient *kubecli.MockKubevirtClient
```

Comment:
Is it still required?
Same for `kubeClient`.

Reply:
Thanks!
i removed it.